.gitignore

```diff
@@ -6,6 +6,7 @@ pom.xml.next
 release.properties
 dependency-reduced-pom.xml
 buildNumber.properties
+~$*
```

Comment:
Let's be conservative in what we add to archetype, especially in terms of these blanket ignore (i.e. that are effective for all directories).